CRM/ACL/BAO/Cache.php

```diff
@@ -109,7 +109,8 @@ public static function store($id, &$cache) {
   }
 
   /**
-   * @param int $id
+   * Delete entries from the civicrm_acl_cache for a specific contact.
+   * @param int $id contact_id
```

Comment:
Can we add a row here - it's an un-enforced (by us) part of the drupal comment spec